scripts/globals/spells/choke.lua

```diff
@@ -48,7 +49,13 @@ function onSpellCast(caster,target,spell)
                 end
                 spell:setMsg(dsp.msg.basic.MAGIC_ENFEEB)
                 local duration = math.floor(ELEMENTAL_DEBUFF_DURATION * resist)
-                target:addStatusEffect(dsp.effect.CHOKE,DOT, 3, ELEMENTAL_DEBUFF_DURATION)
+                if caster:getMainJob() == dsp.job.BLM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.